1.0/docs/start/psk/creating-a-page.md

```diff
@@ -0,0 +1,211 @@
+    <iron-icon icon="book"></iron-icon>
+    <span>Books</span>
+  </a>
+</paper-menu>
+...
+{% endhighlight %}
+
+If you view the app now, you should see your new item in the navigation
+menu, but the link does not point to a valid page yet. Proceed to the 
+next section to do that now.
+
+<!-- image with new nav item -->
+
+## Adding content
+
+In the previous section you added a navigation menu item to enable the 
+user to navigate to a new page. Now, you add the content for that new page.
+
+1. Open `app/index.html` and find the main content. The beginning of the main
+   content section is provided in the code snippet below (some code omitted for
+   brevity).
+
+{% highlight html %}
+<div class="content">
+  <iron-pages attr-for-selected="data-route" selected="{{route}}">
+
+    <section data-route="home">
+      <paper-material elevation="1">
+        <my-greeting></my-greeting>
+
+        <p class="paper-font-subhead">You now have:</p>
+        <my-list></my-list>
+        ...
+      </paper-material>
+    </section>
+
+    <section data-route="users">
+      <paper-material elevation="1">
+        <h2 class="paper-font-display2">Users</h2>
+        <p>This is the users section</p>
+        <a href="/users/Rob">Rob</a>
+      </paper-material>
+    </section>
+    ...
+{% endhighlight %}
+
+* Each page of the application is a `section` element. 
+  * The `data-route` attribute is an identifier for the routing system. 
+  You'll set that up for your new page in the next section.
+* The `paper-material` element creates a card which floats on top of the
+  main content area. All text, images, and other content should always
+  be contained within one of these cards.
+* The `elevation` attribute determines how high the card appears to visually
+  float above the main content area. Experiment by setting it to values between
+  `0` and `1` to see for yourself.
+
+1. Add the following content to the bottom of the main section area.
+
+{% highlight html %}
+<section data-route="books">
+  <paper-material elevation="1">
+    <p>Hello, World!</p>
+  </paper-material>
+</section>
+{% endhighlight %}
+
+Your code should now look like the code snippet below (some code omitted).
+
+{% highlight html %}
+...
+<!-- Main Content -->
+<div class="content">
+  <iron-pages attr-for-selected="data-route" selected="{{route}}">
+    ...
+    <section data-route="contact">
+      <paper-material elevation="1">
+        <h2 class="paper-font-display2">Contact</h2>
+        <p>This is the contact section</p>
+      </paper-material>
+    </section>
+
+    <section data-route="books">
+      <paper-material elevation="1">
+        <p>Hello, World!</p>
+      </paper-material>
+    </section>
+
+  </iron-pages>
+</div>
+...
+{% endhighlight %}
+
+You now have content to link your new navigation item to, but the routing
+system for the Polymer Starter Kit is not quite set up properly yet. Proceed
+to the next section to link your navigation item to your new content.
```

Comment:
Maybe pare down to two sentences here? You now have content to link your new navigation item to. In the 
next section you'll link your navigation item to your new content.